README.md

```diff
@@ -1,6 +1,13 @@
-dp-frontend-renderer
+# dp-frontend-renderer
```

Comment:
Similar to the migration doc - we don't need the `=======` anymore if we are using `#`